### PR TITLE
Upgrade the version of ecto

### PR DESCRIPTION
**Pull request by elpic**
Update the version of ecto and fix the dependency issues with Bcrypt

### CODE REVIEW

**Comment by mustafaturan** [comment]
Hi @elpic 
Thanks for contributing, can you remove elixir 1.4 from the travis.yml and try again?

**Comment by elpic** [comment]
Hi @mustafaturan,

Done :D

File: test/authable/auth_strategies/query_param_test.exs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-defmodule Authable.AuthStrategy.QueryParamTest do
+  defmodule Authable.AuthStrategy.QueryParamTest do
```
Comment (mustafaturan) [inline test/authable/auth_strategies/query_param_test.exs]: extra space

**Comment by mustafaturan** [comment]
Can you squash your commits into 1 commit? Then I will merge and will bump the package

**Comment by tpitale** [comment]
@elpic Can you finish this PR so it can be released?

**Comment by mustafaturan** [comment]
Closed in favor of https://github.com/mustafaturan/authable/pull/116

**Pull request closed without merging**